src/qpdf/jbig2-inl.h

```diff
@@ -107,6 +107,8 @@ class JBIG2StreamFilter : public QPDFStreamFilter {
 
     virtual bool setDecodeParms(QPDFObjectHandle decode_parms) override
     {
+        if (decode_parms.isNull())
+            return false;
```

Comment:
This is not quite correct - should return true instead since the decode parameters are optional for JBIG2 images.